CHANGES.md

```diff
@@ -12,6 +12,7 @@ Apollo 1.10.0
 * [Make Access Key Timestamp check configurable](https://github.com/ctripcorp/apollo/pull/3908)
 * [remove ctrip profile](https://github.com/ctripcorp/apollo/pull/3920)
 * [Remove spring dependencies from internal code](https://github.com/apolloconfig/apollo/pull/3937)
+* [Fix gpg signing issues when deploying to maven repository with github action](https://github.com/apolloconfig/apollo/pull/3950)
```

Comment:
Change log should be what users cared, I don't think we need to include this kind of chore

Reply:
Sounds reasonable, I have deleted it from CHANGES.md